README.md

```diff
@@ -106,13 +106,13 @@ The `IBiometryService` has severals methods.
 
 As of now, this is the list of features available per platform.
 
-| Methods          | iOS     | Android | WinUI   | UWP     |
-| ---------------- | :-----: | :-----: | :-----: | :-----: |
-| `GetCapability`  | ✔ | ✔ | ✔ | ✔ |
-| `ScanBiometry`	 | ✔ | ✔ | ✔ | ✔ | 
-| `Encrypt`        | ✔ | ✔ | ✔ | ✔ |
-| `Decrypt`        | ✔ | ✔ | ❌ | ❌ |
-| `Remove`         | ✔ | ✔ | ✔ | ✔ |
+| Methods          | iOS     | Android | WinUI   | UWP     | Tests   |
```

Comment:
I don't think we should mention tests in this table because it's not a real implementation.